.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: master
+on:
+  push:
+    branches:
+      - master
+permissions:
+  contents: read
+  id-token: write
+concurrency:
+  cancel-in-progress: false
+  group: "${{ github.workflow }}-${{ github.ref }}"
+jobs:
+  upload-aws-backup-elastio-integration:
+    runs-on:
+      - ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: "actions/checkout@v3"
+        with:
+          fetch-depth: '30'
+          token: "${{ secrets.SECRET_TOKEN }}"
+      - name: Check if files were modified
+        id: changes
+        uses: "dorny/paths-filter@v2"
+        with:
+          filters: |
+            filechange:
+              - 'aws-backup-elastio-integration/**'
+      - name: Configure AWS credentials
+        if: "steps.changes.outputs.filechange == 'true'"
+        uses: "aws-actions/configure-aws-credentials@v3"
+        with:
+          aws-region: "${{ secrets.CI_AWS_DEFAULT_REGION }}"
+          role-to-assume: "${{ secrets.CI_AWS_ROLE_ARN}}"
```

Comment:
These could also be `vars`

Reply:
Defined under the job. I think we can place as a global envs when the pipeline will grow.